File: app/api/v1/router.py
"""
API v1 路由聚合

聚合所有v1版本的API端点
"""

from fastapi import APIRouter

from app.api.v1.endpoints import health

# 创建v1版本的路由器
api_router = APIRouter()

# 注册健康检查路由
api_router.include_router(health.router, tags=["健康检查"])

# 在这里添加其他路由
# api_router.include_router(auth.router, prefix="/auth", tags=["认证"])
# api_router.include_router(users.router, prefix="/users", tags=["用户"])

File: src/api/v1/router.ts
import { FastifyPluginAsync } from "fastify";
import { healthRouter } from "./endpoints/health.js";
import { authRouter } from "./endpoints/auth.js";
import { usersRouter } from "./endpoints/users.js";
import { mediaRouter } from "./endpoints/media.js";
import { categoriesRouter } from "./endpoints/categories.js";
import { statsRouter } from "./endpoints/stats.js";

/**
 * API v1 路由聚合
 *
 * 聚合所有 v1 版本的 API 端点
 */
export const apiV1Router: FastifyPluginAsync = async (fastify) => {
  await fastify.register(healthRouter);
  await fastify.register(authRouter, { prefix: "/auth" });
  await fastify.register(usersRouter, { prefix: "/users" });
  await fastify.register(mediaRouter, { prefix: "/media" });
  await fastify.register(categoriesRouter, { prefix: "/categories" });
  await fastify.register(statsRouter, { prefix: "/stats" });
};
